ui/pages/settings/advanced-tab/advanced-tab.component.test.js

```diff
@@ -105,4 +146,24 @@ describe('AdvancedTab Component', () => {
       expect(mockSetStxOptIn).toHaveBeenCalled();
     });
   });
+
+  describe('renderStateLogs', () => {
+    it('should render the toggle button for state log download', () => {
+      const { queryByTestId } = renderWithProvider(<AdvancedTab />, mockStore);
+      const stateLogButton = queryByTestId('advanced-setting-state-logs');
+      expect(stateLogButton).toBeInTheDocument();
+    });
+
+    it('should call exportAsFile when the toggle button is clicked', () => {
+      // const exportSpy = jest.spyOn(exportUtil, 'exportAsFile')
+      // const exportAsFileMocked = jest.mocked(exportAsFile);
+      const { queryByTestId } = renderWithProvider(<AdvancedTab />, mockStore);
+      const stateLogButton = queryByTestId(
+        'advanced-setting-state-logs-button',
+      );
+      fireEvent.click(stateLogButton);
+      expect(exportUtil.exportAsFile).toHaveBeenCalled();
```

Comment:
This test is failing? 
https://github.com/MetaMask/metamask-extension/actions/runs/11040557627/job/30668816032#step:4:7979